src/layers/core/hexagon-layer/hexagon-layer-vertex.glsl

```diff
@@ -0,0 +1,91 @@
+#define SHADER_NAME extruded-hexagon-layer-vs
+
+attribute vec3 positions;
+attribute vec3 normals;
+
+attribute vec3 instancePositions;
+attribute vec4 instanceColors;
+attribute vec3 instancePickingColors;
+
+// Picking uniforms
+// Set to 1.0 if rendering picking buffer, 0.0 if rendering for display
+uniform float renderPickingBuffer;
+uniform vec3 selectedPickingColor;
+
+// projection uniforms
+uniform float mercatorScale;
+uniform mat4 worldMatrix;
+uniform mat4 viewMatrix;
+uniform mat4 worldInverseTransposeMatrix;
+
+// Custom uniforms
+uniform float opacity;
+uniform vec3 invisibleColor;
+uniform float radius;
+uniform float angle;
+uniform float enable3d;
+
+// Lighting constants, this can also be set via uniform
+const vec3 ambientColor = vec3(1., 1., 1.);
+const vec3 pointLocation = vec3(-1., 3., -1.);
+const vec3 pointColor = vec3(1., 1., 1.);
+const vec3 pointSpecularColor = vec3(0.6, 0.6, 0.6);
+const float shininess = 3.;
+const float pointLightAmbientCoefficient = 0.8;
+
+// Result
+varying vec4 vColor;
+
+// a magic number to scale elevations so that 1 unit roughly equals 100 meter
+float elevationScaler = 6000.;
```

Comment:
Does `project_scale` not work here?

If not maybe use a #define `ELEVATION_SCALE` to make it clear that this is a constant.